### DISCUSSION TITLE
Mummy the Resurrection Sheet

### DISCUSSION THREAD

**Discussion by PeterMcD-stack** [JohanFalt/Foundry_WoD20#939] [Q&A] [answered]
Hey, super appreciate all the work you do on this, any chance for a mummy sheet in the future?

**Comment by JohanFalt** [comment, marked as answer]
Will arrive in next version 4.2.

**Comment by JohanFalt** [comment]
![{0B3A0D1A-9D80-4264-A42B-5FA6972F58B2}](https://github.com/user-attachments/assets/be380812-09f7-4d15-af39-84814875a313)

![{BF1A729C-513B-477F-B550-13A34971B51D}](https://github.com/user-attachments/assets/95516891-84ca-469b-995c-68e0614d4f79)

![{841181A3-B4E3-490E-A688-541E65E1989C}](https://github.com/user-attachments/assets/cd3f7b5a-33a0-4219-b454-0e466392a7c5)
